projects/plugins/jetpack/extensions/blocks/donations/amount/save.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import formatCurrency, { CURRENCIES } from '@automattic/format-currency';
+
+/**
+ * External dependencies
+ */
+import { minimumTransactionAmountForCurrency } from '../../../shared/currencies';
+import classnames from 'classnames';
+
+const Save = ( { attributes } ) => {
+	const { disabled, currency, amount } = attributes;
+
+	if ( disabled ) {
+		const className = classnames( 'donations__amount wp-block-button wp-block-button__link', {
+			'donations__custom-amount': disabled,
+		} );
```

Comment:
We don't need the `classnames` utility, as in this case `disabled` is always true.
```js
const className = 'donations__amount wp-block-button wp-block-button__link donations__custom-amount';
```

Otherwise, we could move the `className` and `blockProps` to the top, and save a couple of lines:

```js
const className = classnames( 'donations__amount wp-block-button wp-block-button__link', {
	'donations__custom-amount': disabled,
} );
const blockProps = useBlockProps.save( { className } );

if ( disabled ) {
	return <div { ...blockProps }>
		{ /* ... */ }
	</div>;
}

return <div { ...blockProps } data-amount={ amount }>
	{ /* ... */ }
</div>;
```

Reply:
True, I'll refactor to extract the initialization of className and blockProps to not be dependent on branching conditions.

Reply:
Done.